courses/free-courses-hi.md

```diff
@@ -69,9 +76,16 @@
 
 #### React
 
+* [React Js Tutorials in Hindi](https://youtube.com/playlist?list=PLu0W_9lII9agx66oZnT6IyhcMIbUMNMdt) - CodeWithHarry
+* [React Tutorial in Hindi](https://www.youtube.com/watch?v=RGKi6LSPDLU) - CodeWithHarry
```

Comment:
Not playlist but is a 2h video having timeline as starred comment

![image](https://user-images.githubusercontent.com/3125580/135701713-b8ad88b2-5e54-41e9-b0b4-fe9a3e54396a.png)

Reply:
Shall I remove this video?

Reply:
It's only a note for merging. Maybe Eric tell something about